shared/github_oauth.go

```diff
@@ -100,3 +115,51 @@ func NewGitHubOAuth(ctx context.Context) (GitHubOAuth, error) {
 
 	return &githubOAuthImp{ctx: ctx, conf: oauth, ds: store}, nil
 }
+
+var secureCookie *securecookie.SecureCookie
+
+// GetSecureCookie returns an instance of securecookie.
```

Comment:
This documentation just tells me a shallow version of what this method does, which I can also get just by looking at the return type. It would be more useful to have something like:

```
// GetSecureCookie returns the securecookie instance for wpt.fyi. This instance can
// be used to encode and decode cookies set by wpt.fyi, to avoid forgery attacks.
```